content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -19,49 +19,51 @@ The features described on this page are in beta. Contact your Customer Success M
 - **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous Database
 - **Architecture**: Multi-tenant
 
-## Prerequisites
+## Overview
 
-### Install the Oracle integration
-
-On the [Integrations][1] page in Datadog, install the [Oracle integration][2] for your organization. This will install an [Oracle dashboard][5] in your account which can be used to monitor the performance of your Oracle databases.
-
-### Upgrade the Oracle integration in your Agent
-
-You can skip this step if this is your first time installing the Oracle integration. If you already installed the Oracle integration, migrate the legacy configuration from the `conf.d/oracle.d/` directory to the new integration path located in the `conf.d/oracle-dbm.d/` directory.
-
-Use the following command to migrate the Oracle integration from the legacy integration to the new one:
-
-```shell
-cp /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml
-```
+### Prerequisites
 
-Deactivate the legacy integration:
+Before Database Monitoring can be configured for Oracle, the following prerequisites must be met:
 
-```shell
-mv /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle.d/conf.yaml.bak
-```
+1. An [Agent version](#recommended-agent-version) that supports Oracle monitoring features must be installed.
+    - [Install the Agent](#install-the-agent)
+    - [Upgrade an existing Agent installation](#upgrade-from-a-previous-agent-release)
+2. The Oracle integration must be installed.
+    - [Install the Oracle integration](#install-the-oracle-integration)
+    - [Verify an existing Oracle integration installation](#verify-your-existing-oracle-integration)
 
-Deactivating the legacy integration prevents sending the system metrics twice.
+### Setup
+If the above prerequisites are met, follow the setup instructions for your hosting type:
+{{< partial name="dbm/dbm-setup-oracle" >}}
 
-Since the Agent doesn't require an external Oracle client, remove the `jdbc_driver_path` configuration parameter from the new parameter file `/etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml`.
+## Prerequisite details
 
 ### Install the Agent
 
-To start collecting Oracle telemetry, first install the Datadog Agent.
+#### Host requirements
 
 The Agent does not need to run on the same server nor the same platform as the monitored databases, and can be installed remotely using one of the suggested [setup architectures][10].
```

Comment:
> I think this part is misleading: `The Agent does not need to run on the same server nor the same platform as the monitored databases ...`
> 
> since in self-hosted setups, the Agent is in fact running on the same server

Not necessarily - the Agent still can be running on a different server. For example, it is possible to install a single agent monitoring different databases on different hosts. In fact, some customers explicitly asked for that.